templates/eap-cd-amq-persistent-s2i.json

```diff
@@ -520,7 +520,7 @@
                         "from": {
                             "kind": "ImageStreamTag",
                             "namespace": "${IMAGE_STREAM_NAMESPACE}",
-                            "name": "eap-cd-openshift:19"
+                            "name": "eap-cd-openshift:20"
```

Comment:
should we use `eap-cd-openshift:latest` so we don't have to update the templates for every new release?

Reply:
In the past we've always be cautious about using :latest as it could end up with a customer being forced to update over a release (say from 7.2 to 7.3 or something like that, though those have different names as well). I'm not sure this is a concern with CD though, so yeah I think we could do this.

The only other thing that might be odd is you could have a mix of say :20 builder with :19 runtime and not immediately know until you compared the versions.

I think we should do it though.

Reply:
@jmesnil Updated, other than changing the version if we actually change the templates in some way, I think they would no longer need to be changed every time.